src/test/java/org/apache/commons/collections4/iterators/CollatingIteratorTest.java

```diff
@@ -16,12 +16,14 @@
  */
 package org.apache.commons.collections4.iterators;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.List;
-
 import org.apache.commons.collections4.comparators.ComparableComparator;
+import org.junit.Test;
+
+import java.lang.reflect.Array;
+import java.util.*;
```

Comment:
This looks like you're using IntelliJ with the default settings. I've had to disable the change to `import package.name.*` for 5 or more from the same location. I would try to turn that off. Further it may not run on save as some folks suggested in https://github.com/apache/commons-text/pull/56 I think it runs when you do a file format command or "reorganize imports" command. Regardless I'll try to change it back.